tox.ini

```diff
@@ -34,6 +34,28 @@ deps =
 commands =
    pytest {[vars]tst_path}integration -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs}
 
+[testenv:integration-charm]
+description = Run integration tests
+deps =
+    ipdb==0.13.9
+    juju==3.1.0.1
```

Comment:
Is this correct?

Reply:
The versions of Juju available for testing with tox include `2.9.44.1, 2.9.45.0, 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4, 3.1.0.1, 3.1.2.0, 3.2.0.0, 3.2.0.1, 3.2.2, 3.2.3.0` as we are using the 3.1.6 controllers I think this is appropriate of the choices to be testing with.